src/components/VmDetails/cards/UtilizationCard/UtilizationCharts/BarChart.js

```diff
@@ -41,13 +41,13 @@ const BarChart = ({ data, additionalLabel, thresholdWarning, thresholdError, id,
             }}
             labelComponent={<CustomLabel offsetX={offsetX} label={additionalLabel} />}
             data={data}
-            {...rest}
+            labels={({ datum }) => `${msg.utilizationCardLegendUsed()} ${datum.y}%  (${datum.x})`}
```

Comment:
So just FYI, the percent symbol location and spacing can change per locale, so in ui-extensions we have a message like this:
https://github.com/oVirt/ovirt-engine-ui-extensions/blob/8a0269401791f58667d93d43d3a1728db08f9457/src/intl/messages.js#L89-L93

I know you just reused the label, but that formatting should be translatable.